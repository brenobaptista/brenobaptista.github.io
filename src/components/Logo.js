import React from 'react';
import Logo from '../images/favicon.png';

const logo = () => (
  <div className="logo">
    <img src={Logo} alt="logo" />
  </div>
);

export default logo;
