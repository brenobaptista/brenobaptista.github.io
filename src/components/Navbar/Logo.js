import React from 'react'
import Favicon from '../../images/favicon.png'

const Logo = () => (
  <div className='logo'>
    <img
      src={Favicon}
      alt='logo'
    />
  </div>
)

export default Logo
