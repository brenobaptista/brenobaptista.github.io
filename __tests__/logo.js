import React from 'react'
import { render } from '@testing-library/react'
import Logo from '../src/components/Navbar/Logo'

test('matches snapshot with desktop logo', () => {
  const { asFragment } = render(<Logo />)

  expect(asFragment(<Logo />)).toMatchSnapshot()
})

test('matches snapshot with mobile logo', () => {
  const { asFragment } = render(<Logo mobile />)

  expect(asFragment(<Logo mobile />)).toMatchSnapshot()
})
