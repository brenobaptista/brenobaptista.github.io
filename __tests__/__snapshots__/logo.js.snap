// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot with desktop logo 1`] = `
<DocumentFragment>
  <div
    class="logo"
  >
    <img
      alt="logo"
      src="test-file-stub"
    />
  </div>
</DocumentFragment>
`;

exports[`matches snapshot with mobile logo 1`] = `
<DocumentFragment>
  <div
    class="logo"
  >
    <img
      alt="logo"
      src="test-file-stub"
    />
  </div>
</DocumentFragment>
`;
